webapp/src/plugin.jsx

```diff
@@ -18,6 +18,14 @@ export default class Plugin {
 
             registry.registerRootComponent(CreateIssueModal);
             registry.registerPostDropdownMenuComponent(CreateIssuePostMenuAction);
+
```

Comment:
1/5 This seems to be the wrong place to do it

Reply:
I agree, it doesn't seem the right place. But:
1. I can't think of a more appropriate place
1. plugin-demo has a lot more than that in its initialize function. :)